dev/ToastNotifications/ToastNotificationManager.cpp

```diff
@@ -131,14 +134,14 @@ namespace winrt::Microsoft::Windows::ToastNotifications::implementation
     // For other scenarios the API throws
     winrt::Windows::Foundation::IAsyncOperation<winrt::Microsoft::Windows::ToastNotifications::ToastProgressResult> ToastNotificationManager::UpdateToastProgressDataAsync(winrt::Microsoft::Windows::ToastNotifications::ToastProgressData const& data, hstring const& tag, hstring const& group)
     {
-        if ((tag == winrt::hstring(L"") && group == winrt::hstring(L"")) || (tag == winrt::hstring(L"") && group != winrt::hstring(L"")))
+        if (tag == winrt::hstring(L""))
         {
             co_return winrt::ToastProgressResult::NotificationNotFound;
```

Comment:
I don't think this outcome is the right one. We can post toasts with no tag. But I do agree this scenario is not valid for updating progress data. This case seems more like a developer mistake. I would throw here as E_INVALIDARG.

Reply:
I just noticed there was a mini thread discussion on it. Take my comment above as my two cents on it 😊.